sympy/physics/vector/frame.py

```diff
@@ -727,50 +727,35 @@ def orient_explicit(self, parent, dcm):
         UserWarning
             If the orientation creates a kinematic loop.
 
-        Examples
-        ========
-
-        Setup variables for the examples:
+        """
+        _check_frame(parent)
+        # amounts must be a Matrix type object
+        # (e.g. sympy.matrices.dense.MutableDenseMatrix).
+        if not isinstance(dcm, MatrixBase):
+            raise TypeError("Amounts must be a SymPy Matrix type object.")
```

Comment:
This check is also done in `orient_dcm`, so it can be removed here.

Reply:
```
Traceback (most recent call last):
  File "z:\sympy\sympy\physics\vector\tests\test_frame.py", line 400, in test_reference_frame
    raises(TypeError, lambda: A.orient(B, 'DCM', 0))
  File "z:\sympy\sympy\testing\pytest.py", line 116, in raises
    code()
  File "z:\sympy\sympy\physics\vector\tests\test_frame.py", line 400, in <lambda>
    raises(TypeError, lambda: A.orient(B, 'DCM', 0))
                              ^^^^^^^^^^^^^^^^^^^^^
  File "z:\sympy\sympy\physics\vector\frame.py", line 1188, in orient
    self.orient_explicit(parent, amounts)
  File "z:\sympy\sympy\physics\vector\frame.py", line 737, in orient_explicit
    self.orient_dcm(parent, dcm.T)
                            ^^^^^
AttributeError: 'int' object has no attribute 'T'
```


* When I ran the test case locally, an error occurred. I suggest checking if the parent and dcm are ready before using them in the `orient_dcm` function.

Reply:
That is a good one. Thank you for testing.